kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/URLUtils.java

```diff
@@ -150,4 +152,7 @@ public static boolean isValidURL(String url) {
     }
   }
 
+  public static String encodeCommand(String cmd) throws UnsupportedEncodingException {
```

Comment:
@karthikmg A utility method should be usable anywhere required. The name `encodeCommand` suggests that its sole purpose is to encode commands. But its actually a general utilitilty function which can be used anywhere suitable.

Also, mind the fact that this function encodes a Url here, so, any string which resembles a Url can be an argument for his function. I understand that in this particular use case, we're encoding a command string, but the function isn't restricted to this use case. It can accept any string here.

The name `encodeToUTF` would make much more sense.

``` encodeToUTF(String url) { ... }```

and then calling it like 

```
URLUtils.encodeToUTF(cmd)
```

@rohanKanojia  correct me if I'm wrong

Reply:
@dev-gaur 
Pushed the refactoring changes as per your review.
Please let me know where to add the test case. Should I add test case for URLUtils.encodeToUTF() or exec() in the PodOperationsImpl?

Also, I made changes to Changelog.md file and pushed them before the above refactoring of method name. But CI build is failing. Could you please let me know what the issue is so that I can fix it?